packages/vuetify/src/components/VTreeview/VTreeview.ts

```diff
@@ -68,6 +68,21 @@ export default mixins(
       type: Array,
       default: () => ([])
     } as PropValidator<NodeArray>,
+    search: String,
+    customFilter: {
+      type: Function,
+      default: (items: any[], search: string, idKey: string, textKey: string, childrenKey: string) => {
+        const excluded = new Set()
+
+        if (!search) return excluded
+
+        for (let i = 0; i < items.length; i++) {
+          filterTreeItems(items[i], search, idKey, textKey, childrenKey, excluded)
+        }
+
+        return excluded
+      }
+    },
```

Comment:
`} as PropValidator<(items: any[], search: string, idKey: string, textKey: string, childrenKey: string) => Set<any>>`

And then I think you might be able to remove the types from the default so it isn't duplicated.

Reply:
What does this solve? I toyed around with it and it didn't work